muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/sass/elements/application-panel.scss

```diff
@@ -51,12 +52,18 @@
   }
 }
 
-.application-panel__header-title {
+.application-panel__header-title-container {
```

Comment:
We seem to have quite colorful way to name headers within different blocks: 

- application-panel__header
- application-panel__header-title
- application-panel__header-title-container
- application-list__header
- application-list__header-title

and of course this same continues with sub-panels too: application-sub-panel__header but without any other variations currently.

All of those seem to have only text node as a child or they might have html child elements. We should use __header and __header-container (which seem to be same thing after all) when we know we have html child nodes and __header-title should be used when only text node is placed as a child. 

Right now it seems a bit confusing when to use what className or I may be missing the point of these classNames completely.

Reply:
Yes, we need to discuss this.